### ISSUE TITLE
[1.3 rc.2 ] Can't find npm module 'meteor-deque'

### ISSUE THREAD

**Issue by lyquocnam** [meteor/meteor#6500]
***2 Upvotes*** this release install successfully but still error:
` Error: Can't find npm module 'meteor-deque'. Did you forget to call 'Npm.depends' in package.js within the 'meteor' package?`

windows 10 x64

**Comment by seifip** [comment]
Same here, Windows 10 x64

**Comment by redwards1966** [comment]
+1, Windows 10 x64 also

**Comment by treylon** [comment]
+1 Windows 10 x64

**Comment by santiagopuentep** [comment]
Happening at least since beta 16.

**Comment by jmwierzbicki** [comment]
I just updated meteor to rc-2, and same thing happened - Win10 x64

**Comment by benjamn** [comment]
This is my top priority right now, definitely a real bug, and definitely Windows-specific.

**Comment by gsabran** [comment]
I'm having this bug when deploying to an ubuntu machine as well, when using `Npm.require` (not sure I'm supposed to do so, but the normal `import` was failing) Let me know if you want more details

**Comment by benjamn** [comment]
@gsabran are you using `Npm.require` for a package in a local `node_modules` directory (in your app), or for something installed with `Npm.depends`?

**Comment by gsabran** [comment]
yes from `node_modules`
I could not load it like other npm packages with `import` on beta11 and it worked with `Npm.require` on my local machine (a mac), but failed with `Can't find npm module 'meteor-deque'...` when deploying on ubuntu.
Now with rc2 it seems to work with `import` (I still have issues deploying so not 100% sure about that but those issues seem to be related to something else)
The package is juice (https://www.npmjs.com/package/juice)

**Comment by benjamn** [comment]
This will be fixed in the next RC (`1.3-rc.3` most likely).

**Comment by gsabran** [comment]
Cool. I just want to confirm that I'm consistently having this bug on ubuntu with rc2 when having `import juice from 'juice';` in my server code (juice being in my `package.json`) The bug disappears when I remove the import (ie it's probably not just windows) It runs fine locally.

**Comment by benjamn** [comment]
@gsabran thanks for the heads up. If you have a minute to try running Meteor [from a checkout](https://forums.meteor.com/t/meteor-1-3-early-beta-now-available/14723/356?u=benjamn), you can see if the latest commits I pushed (9021ba8756777d4e9643afa379c66705f8eb71f8 and ff023e9c3777c907c4591f9a8485ca9aa7bd45bd) fix your problems, a bit before the next RC is released.

**Comment by gsabran** [comment]
would love to do if that helps (else I can wait a bit :) ) I'm using meteor up to deploy. Do you know if it's good enough to fetch the latest release locally, or if the build will fetch meteor independently, looking at the version in `./meteor/release`? Also, where should I run `git clone --branch release-1.3 https://github.com/meteor/meteor.git
./meteor/meteor` from? my meteor project?

**Comment by nathantreid** [comment]
I'm happy to report that the latest checkout of Meteor 1.3 is working for me on Windows 10 x64! Super excited to update from beta.12!

**Comment by lucidprogrammer** [comment]
@benjamn 
Windows 10 64. This is the release-1.3 checkout - fails

```
C:\Users\projects\meteor>.\meteor --version
Unreleased, running from a checkout at ff023e9 (HEAD -> release-1.3,
origin/release-1.3)
```

```
C:\Users\projects\meteor>.\meteor create test
Errors prevented isopacket load:

While loading isopacket `constraint-solver`:
error: Can't load npm module 'meteor-deque' from packages/meteor.js. Check
your Npm.depends().


C:\Users\projects\meteor\dev_bundle\lib\node_modules\meteor-promise\promise_server.js:116
      throw error;
            ^
Error: isopacket load failed?
    at loadIsopacketFromDisk (/tools\tool-env\isopackets.js:286:11)
    at /tools\tool-env\isopackets.js:88:23
    at Object.exports.noYieldsAllowed (/tools\utils\fiber-helpers.js:37:14)
    at Object.load (/tools\tool-env\isopackets.js:80:23)
    at ProjectContext._buildResolver (/tools\project-context.js:704:22)
    at /tools\project-context.js:489:29
    at /tools\utils\buildmessage.js:359:18
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at /tools\utils\buildmessage.js:352:34
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at /tools\utils\buildmessage.js:350:23
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (/tools\utils\buildmessage.js:324:26)
    at /tools\project-context.js:488:20
    at /tools\packaging\catalog\catalog.js:100:5
    at /tools\utils\buildmessage.js:271:13
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at /tools\utils\buildmessage.js:264:29
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at /tools\utils\buildmessage.js:262:18
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at /tools\utils\buildmessage.js:253:23
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at Object.capture (/tools\utils\buildmessage.js:252:19)
    at Object.catalog.runAndRetryWithRefreshIfHelpful (/tools\packaging\catalog\catalog.js:99:31)
    at ProjectContext._.extend._resolveConstraints (/tools\project-context.js:487:13)
    at /tools\project-context.js:278:9
    at /tools\utils\buildmessage.js:359:18
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at /tools\utils\buildmessage.js:352:34
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at /tools\utils\buildmessage.js:350:23
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (/tools\utils\buildmessage.js:324:26)
    at ProjectContext._completeStagesThrough (/tools\project-context.js:268:18)
    at /tools\project-context.js:260:12
    at Function.run (/tools\tool-env\profile.js:489:12)
    at ProjectContext.prepareProjectForBuild (/tools\project-context.js:259:13)
    at /tools\cli\commands.js:721:20
    at /tools\utils\buildmessage.js:271:13
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at /tools\utils\buildmessage.js:264:29
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at /tools\utils\buildmessage.js:262:18
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at /tools\utils\buildmessage.js:253:23
    at [object Object].withValue (/tools\utils\fiber-helpers.js:89:14)
    at Object.capture (/tools\utils\buildmessage.js:252:19)
    at Object.main.captureAndExit (/tools\cli\main.js:279:29)
    at Command.buildCommands.minArgs [as func] (/tools\cli\commands.js:704:8)
    at /tools\cli\main.js:1408:23
```

**Comment by benjamn** [comment]
This should have been fixed by [`1.3-rc.3`](https://github.com/meteor/meteor/issues/6266#issuecomment-198569509).

@lucidprogrammer perhaps try it again with [a more recent release](https://github.com/meteor/meteor/issues/6266#issuecomment-198997482)?

**Comment by lucidprogrammer** [comment]
yes all good it is working well, tks @benjamn
